tools/change-registry.sh

```diff
@@ -118,6 +119,16 @@ function parse_args() {
     exit 1
   fi
```

Comment:
Also please fix [this](https://github.com/deckhouse/deckhouse/pull/3499/files#diff-8bf4756771579a6415a1f1eef7cc430258b62a822e0470732b116b6c4c45d631R105) validator for support images paths with port

Reply:
https://github.com/deckhouse/deckhouse/blob/8efbfc6b68208d461c839379c62ffcc7d1f965f9/tools/change-registry.sh#L105

Reply:
And one thing. Do you check changing registry operation with CA file?